corehq/apps/hqadmin/management/utils.py

```diff
@@ -41,10 +41,19 @@ def get_deploy_email_message_body(environment, user, compare_url):
 
 
 def _get_pr_numbers(last_deploy, current_deploy):
-    repo = GitHub().repository('dimagi', 'commcare-hq')
-    comparison = repo.compare_commits(last_deploy, current_deploy)
-
-    pr_numbers = [int(re.search(r'Merge pull request #(\d+)', repo_commit.commit.message).group(1)) for repo_commit in [repo_commit for repo_commit in comparison.commits if repo_commit.commit.message.startswith('Merge pull request')]]
+    repo = Github().get_organization('dimagi').get_repo('commcare-hq')
+    last_deploy_sha = repo.get_commit(last_deploy).sha
+    current_deploy_sha = repo.get_commit(current_deploy).sha
+    comparison = repo.compare(last_deploy_sha, current_deploy_sha)
+
+    merge_commits = [
+        repo_commit for repo_commit in comparison.commits
+        if repo_commit.commit.message.startswith('Merge pull request')
+    ]
+    pr_numbers = [
+        int(re.search(r'Merge pull request #(\d+)', repo_commit.commit.message).group(1))
+        for repo_commit in merge_commits
```

Comment:
nit: could change this line to
```py
        for repo_commit in comparison.commits
        if repo_commit.commit.message.startswith('Merge pull request')
```
and eliminate `merge_commits = [...]` lines above

Reply:
good call; updated